CONTRIBUTING.md

```diff
@@ -58,32 +58,22 @@ by [Vincent Driessen](https://nvie.com/posts/a-successful-git-branching-model/):
 6. Submit a pull request (PR)
     - Follow the [steps](https://help.github.com/en/articles/creating-a-pull-request) of the github help to create the PR.
     - Please note that you PR should be directed from your branch (for example `myfeature`) towards the branch `dev`
+    - To make reviewing easier, briefly describe the changes you have made in the pull request and summarise the discussion and conclusions in the associated issue. 
```

Comment:
I think I used American English convention whenever I wrote, but that's because I spent some time in Canada. I have no preference for one in particular, however I think it look nicer to have coherence. I let you choose which convention we'll abide by in the repo as you are a native speaker and I am not :)

Reply:
That's a good point, I did not pay attention to this. I do not know which is more preferred within Germany. Perhaps, let's address that as part of updating the wiki to include more comprehensive documentation, we can include a guideline about language usage?

Reply:
That is a good idea :)